confluence.go

```diff
@@ -103,7 +127,6 @@ func (api *API) GetAuditRecords(params AuditParameters) (*AuditRecordCollection,
 		"GET", "/rest/api/audit",
 		params, result, nil,
 	)
-
```

Comment:
Please do not change code formatting for other parts of code.

Reply:
Done